OWNERS

```diff
@@ -0,0 +1,10 @@
+approvers:
+- calebevans
+- myakove
+- rnetser
+- rujutashinde
```

Comment:
please remove; should only be a reviewer

Reply:
Which one? I see four names here.

Reply:
I guess the name in the line that the comment add for.